TerrorConsole/Assets/_Main/Scripts/Input/IInputSource.cs

```diff
@@ -4,6 +4,7 @@ namespace TerrorConsole
 {
     public interface IInputSource
     {
+        bool ActionButton1 { get; }
```

Comment:
Needs to have a specific name and be an ´Action´ type, so it becomes an event.
´Action OnActivateCloseObject { get; }´